samples/cellular/location/boards/thingy91x_nrf9151_ns.conf

```diff
@@ -0,0 +1,66 @@
+#
```

Comment:
Wrong place but I'll create conversation here.

What about other samples supporting WiFi positioning (modem_shell and nrf_cloud_multi_service?
I don't know if we need them but just wanting to understand the scope of limiting this to two apps/samples.

Reply:
Currently, we're adding the Thingy to sample.yaml files once they are confirmed working.
The other two are already supported, and could get wifi added as well.